src/functionalTest/resources/application.yaml

```diff
@@ -14,6 +14,7 @@ azure:
   servicebus:
     envelopes:
       max-delivery-count: 10
+      connection-string: ${ENVELOPES_QUEUE_READ_CONN_STRING}
```

Comment:
This cane be defined after this  
https://github.com/hmcts/bulk-scan-orchestrator/blob/master/src/functionalTest/resources/application.yaml#L8-L10

Reply:
done